tf_keras/callbacks.py

```diff
@@ -629,8 +629,8 @@ class Callback:
     inference lifecycle.
 
     To create a custom callback, subclass `keras.callbacks.Callback` and
-    override the method associated with the stage of interest. See
-    https://www.tensorflow.org/guide/tf_keras/custom_callback for more
+    override the method associated with the stage of interest. See the 
+    [Custom callback](https://www.tensorflow.org/guide/tf_keras/custom_callback) for more
```

Comment:
3 small things:
- can you say "custom callback guide" instead of "custom callback"
- can you change the URL to https://www.tensorflow.org/guide/keras/custom_callback (the other one doesn't actually work)
- can you reformat the line so that it doesn't exceed 80 characters

Thanks!

Reply:
> 3 small things:
> 
> * can you say "custom callback guide" instead of "custom callback"
> * can you change the URL to https://www.tensorflow.org/guide/keras/custom_callback (the other one doesn't actually work)
> * can you reformat the line so that it doesn't exceed 80 characters
> 
> Thanks!


<br>
@hertschuh , Updated the given URL. Adding 'guide' in URL text will exceed the 80 chars limit. So I have not added the 'guide' text.